app/src/main/java/com/futurice/freesound/feature/search/SearchActivity.java

```diff
@@ -180,4 +201,17 @@ private void addSearchFragment() {
                                    .commit();
     }
 
+    @Override
+    public void onPause() {
+        super.onPause();
+        dismissSnackbar();
+    }
+
+    private void showSnackbar(@NonNull final CharSequence charSequence) {
+        get(searchSnackbar).showNewSnackbar(coordinatorLayout, charSequence);
```

Comment:
You probably have a nullability warning for `coordinatorLayout`, wrap it with a `get(coordinatorLayout)` call: we want to catch nulls as soon as possible.